### ISSUE TITLE
A11y_Activity Type_Keyboard: In high contarst black/white mode user is not able to see focus visibility while navigating through the multi-select ComboBox

### ISSUE THREAD

**Issue by rishabhsharma0** [microsoft/fluentui#15217]
### Environment Information

- **Package version(s)**: (fill this out)
- **Browser and OS versions**: Microsoft Edge 85.0.552.1 (Official build) dev (64-bit) and Microsoft Windows Version 2004 (OS Build 19041.329)

### Describe the issue:
The issue is coming in multiselect ComboBox. During High Contrast Mode, while Navigating through the list items in multiselect ComboBox dropdown , user is not able to see the focus Visibility.

Steps:
1.Turn on high contrast black/white mode from system settings
2. Head Over to multiselect ComboBox.
3.Navigate through the list items and verify that user is not able to see the focus visibility in Edge Browser

### Please provide a reproduction of the issue in a codepen:
https://codepen.io/rishabhsharma0/pen/VwagBYe

#### Actual behavior:

In high contrast black/white mode user is not able to see focus visibility while navigating through the multiselect ComboBox

#### Expected behavior:

In high contrast black/white mode user should be able to see focus visibility while navigating through the multiselect ComboBox

**Comment by paulgildea** [comment]
@rishabhsharma0 Thanks for the issue. Could you let us know what version of the library you found the issue in? That's helpful because we've been working on fixes in this area recently.

@xugao FYI since this is high contrast.

@miroslavstastny - Try to reproduce this issue with the current version of `ComboBox` on the website.

**Comment by rishabhsharma0** [comment]
version => "office-ui-fabric-react": "7.105.12"
